docs/scos/dev/feature-integration-guides/202304.0/glue-api/dynamic-data-api/dynamic-data-api-integration.md

```diff
@@ -0,0 +1,363 @@
+| DynamicEntityFieldCondition | class | created | src/Generated/Shared/Transfer/DynamicEntityFieldConditionTransfer.php |
+| DynamicEntityFieldDefinition | class | created | src/Generated/Shared/Transfer/DynamicEntityFieldDefinitionTransfer.php |
+| DynamicEntityFieldValidation | class | created | src/Generated/Shared/Transfer/DynamicEntityFieldValidationTransfer.php |
+| Error | class | created | src/Generated/Shared/Transfer/ErrorTransfer.php |
+| GlueError | class | created | src/Generated/Shared/Transfer/GlueErrorTransfer.php |
+| GlueFilter | class | created | src/Generated/Shared/Transfer/GlueFilterTransfer.php |
+| GlueRequest | class | created | src/Generated/Shared/Transfer/GlueRequestTransfer.php |
+| GlueResource | class | created | src/Generated/Shared/Transfer/GlueResourceTransfer.php |
+| GlueResponse | class | created | src/Generated/Shared/Transfer/GlueResponseTransfer.php |
+| Pagination | class | created | src/Generated/Shared/Transfer/PaginationTransfer.php |
+
+{% endinfo_block %}
+
+### 4) Add translations
+
+Append glossary according to your language configuration:
+
+**src/data/import/glossary.csv**
+
+```yaml
+dynamic_entity.validation.invalid_data_format,"Invalid or missing data format. Please ensure that the data is provided in the correct format. Example request body: {'data':[{...},{...},..]}",en_US
+dynamic_entity.validation.invalid_data_format,"Ungültiges oder fehlendes Datenformat. Stellen Sie bitte sicher, dass die Daten im richtigen Format bereitgestellt werden. Beispiel Anforderungskörper: {'data':[{...},{...}]}",de_DE
+dynamic_entity.validation.persistence_failed,"Failed to persist the data. Please verify the provided data and try again.",en_US
+dynamic_entity.validation.persistence_failed,"Das Speichern der Daten ist fehlgeschlagen. Bitte überprüfen Sie die bereitgestellten Daten und versuchen Sie es erneut.",de_DE
+dynamic_entity.validation.persistence_failed_duplicate_entry,"Failed to persist the data. Please verify the provided data and try again. Entry is duplicated.",en_US
+dynamic_entity.validation.persistence_failed_duplicate_entry,"Das Speichern der Daten ist fehlgeschlagen. Bitte überprüfen Sie die bereitgestellten Daten und versuchen Sie es erneut. Eintrag is doppelt vorhanden.",de_DE
+dynamic_entity.validation.entity_does_not_exist,"The entity could not be found in the database.",en_US
+dynamic_entity.validation.entity_does_not_exist,"Die Entität konnte in der Datenbank nicht gefunden werden.",de_DE
+dynamic_entity.validation.required_data_is_missing_or_forbidden,"Some required data is missing or provided data cannot be modified. Please verify the request and try again.",en_US
+dynamic_entity.validation.required_data_is_missing_or_forbidden,"Einige erforderliche Daten fehlen oder die bereitgestellten Daten können nicht geändert werden. Bitte überprüfen Sie die Anfrage und versuchen Sie es erneut.",de_DE
```

Comment:
Customers should be able based on the response modify(fix) the data they send, and make the request one more time.
This message is entirely unclear. Please fix it here, and in the implementation.